hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
```

Comment:
Indentation is inconsistent; ensure proper indentation for readability and maintainability.